aws/logs_monitoring/README.md

```diff
@@ -96,13 +97,25 @@ If the test "succeeded", you are all set! The test log will not show up in the p
 
 For S3 logs, there may be some latency between the time a first S3 log file is posted and the Lambda function wakes up.
 
-## 6. (optional) Multiline Log support for s3
+## 6. (optional) Scrubbing / Redaction rules
+
+Multiple scrubbing options are available.  `REDACT_IP` and `REDACT_EMAIL` match against hard-coded patterns, while `DD_CUSTOM_SCRUBBING_RULE` allows users to supply a custom regular expression.
+
+- To use `REDACT_IP`, add it as an environment variable and set the value to `true`.  
+    - Text matching `\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}` will be replaced with `xxx.xxx.xxx.xxx`.
+- To use `REDACT_EMAIL`, add it as an environment variable and set the value to `true`.
+	- Text matching `[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+` will be replaced with `xxxxx@xxxxx.com`.
+- To use `DD_CUSTOM_SCRUBBING_RULE`, add it as a environment variable, and supply a regular expression as the value.
+    - Text matching the user-supplied regular expression will be replaced with `xxxxx`, by default. 
+    - To customize the replacement text, change the `xxxxx` to the desired text in the `ScrubbingRuleConfig` for the `DD_CUSTOM_SCRUBBING_RULE` [here](https://github.com/DataDog/datadog-serverless-functions/blob/master/aws/logs_monitoring/lambda_function.py#L80).
```

Comment:
I think we should try to avoid asking customers to change the lambda code, they should be able to customise the lambda just using environment (configuration) parameters. We could for example add another parameter for the replacement text. WDYT ?

Reply:
:+1